images/shared-images-controller/main.go

```diff
@@ -79,15 +79,21 @@ func pullRequiredImages(ctx context.Context, tag string) error {
 
 	for _, version := range versions {
 		log.Infoln("Kubevirt Provider version: ", version)
+
 		name := fmt.Sprintf("%sk8s-%s:%s", kubevirtci_repo, version, tag)
+		// k8s-1.26 kubevirtci provider was renamed to k8s-1.26-centos9
+		// to track the change to CentOS Stream 9
+		if version == "1.26" {
```

Comment:
This fixes the issue for exactly this version.

How about we first query quay about what provider versions are there and then just pull them all? This way we would never have this issue occur again, regardless of what suffix is present? WDYT?

Reply:
OK, we might take this as an improvement for the future.